docs/development/building.md

```diff
@@ -1,36 +1,42 @@
-# Akri Build
+# Building
+
 Building Akri, whether locally or in the automated CI builds, leverages the same set of `make` commands.
 
-In essence, Akri components can be thought of as:
-1. Runtime components
-    1. Rust code: containers based on Rust code are built using `Cargo cross` and subsequent `docker build` commands include the cross-built binaries.
-        > Note: For Rust code, build/Dockerfile.* does NOT run `cargo build`, instead they simply copy cross-built binaries into the container
-    2. Other code: these containers can be .NET or python or whatever else ... the build/Dockerfile.* must do whatever building is required.
-2. Intermediate components: these containers are used as part of the build process and are not used in production explicitly
+In essence, Akri components can be thought of as: 1. Runtime components 1. Rust code: containers based on Rust code are built using `Cargo cross` and subsequent `docker build` commands include the cross-built binaries.
```

Comment:
markdown's automatic list numbering doesn't seem to work when the list is part of a paragraph.

i think the numbers have to be incremented manually or the list needs to be separated as it initially was.

Reply:
this seems true for all of the numbered lists in this page.